main/EDInitMod.F90

```diff
@@ -384,51 +429,107 @@ subroutine init_patches( nsites, sites, bc_in)
 
      else
 
-        !FIX(SPM,032414) clean this up...inits out of this loop
+        allocate(recall_older_patch)
         do s = 1, nsites
-
            ! Initialize the site-level crown area spread factor (0-1)
            ! It is likely that closed canopy forest inventories
            ! have smaller spread factors than bare ground (they are crowded)
            sites(s)%spread     = init_spread_near_bare_ground
-
-           allocate(newp)
-
-           newp%patchno = 1
-           newp%younger => null()
-           newp%older   => null()
-
-           sites(s)%youngest_patch => newp
-           sites(s)%youngest_patch => newp
-           sites(s)%oldest_patch   => newp
-
-           ! make new patch...
-
-           call create_patch(sites(s), newp, age, area, primaryforest)
-           
-           ! Initialize the litter pools to zero, these
-           ! pools will be populated by looping over the existing patches
-           ! and transfering in mass
-           do el=1,num_elements
-              call newp%litter(el)%InitConditions(init_leaf_fines=0._r8, &
+          if(hlm_use_nocomp.eq.itrue)then
+           no_new_patches = numpft
```

Comment:
no_new_patches is short for number of new patches.  is the no_ part latin?

Reply:
or is no_ for no_comp?

Reply:
Yes, the the latin-y one...